Packs/FlashpointFeed/IndicatorFields/incidentfield-Flashpoint-Attribute-ID.json

```
@@ -0,0 +1,28 @@
+{
+    "id": "indicator_flashpointattributeid",
+    "version": -1,
+    "modified": "2021-07-14T18:05:51.430463334+05:30",
+    "name": "Flashpoint Attribute ID",
+    "ownerOnly": false,
+    "description": "Unique Flashpoint ID used to identify an attribute.",
+    "cliName": "flashpointattributeid",
+    "type": "shortText",
+    "closeForm": false,
+    "editForm": true,
+    "required": false,
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 2,
+    "hidden": false,
+    "associatedToAll": true,
```

```suggestion
    "associatedToAll": false,
    "associatedTypes": [
        ""Flashpoint Indicator""
    ],
```
The same change in all other Indicator Fields